.env.dev

```diff
@@ -10,4 +10,5 @@ REACT_APP_SENTRY_ENV=development
 SENTRY_AUTH_TOKEN="db67ae12222811ecabd2564b6436d30d"
 REACT_APP_SENTRY_RELEASE="0.7.0"
 
-REACT_APP_MAINTENANCE=false
\ No newline at end of file
+REACT_APP_MAINTENANCE=false
+REACT_APP_DISABLE_CONNECT_WITH_SUBSTRATE_BUTTON=true
```

Comment:
Rename to `REACT_APP_CONNECT_WITH_SUBSTRATE_WALLET_ENABLED`
Put "false" to all envs.

Reply:
Updated